src/main/java/duke/Event.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+public class Event extends Task{
+    protected String time;
+    protected String type;
+
+    public Event(String information,String time,String type){
```

Comment:
Should a whitespace be placed before the curly bracket, and after the commas? I noticed the same issues of whitespaces, commas and curly brackets in several other places too.